gcs/doc.md

```diff
@@ -0,0 +1,147 @@
+			return
+		}
+		fmt.Println(err.(bifrost.Error).Code(), err)
+		return
+	}
+	defer bridge.Disconnect()
+	fmt.Printf("Connected to %s\n", bridge.Config().Provider)
+}
+```
+And that's it! You have now mounted a Bifrost bridge to your GCS account and can start uploading files using Bifrost.
+
+## Shipping a file to Google Cloud Storage via the rainbow bridge
+Now that you have mounted a Bifrost bridge to GCS, you can use Bifrost to upload files to GCS using the following code:
+```go
+// Upload a file
+uploadedFile, err := bridge.UploadFile(bifrost.File{
+	Path:     "../shared/image/aand.png",
+	Filename: "a_and_ampersand.png",
+	Options: map[string]interface{}{
+		bifrost.OptMetadata: map[string]string{
+			"originalname": "aand.png",
+		},
+	},
+})
+if err != nil {
+	fmt.Println(err)
+	return
+}
+fmt.Printf("Uploaded file: %s to %s\n", uploadedFile.Name, uploadedFile.Preview)
+```
+As you can see, uploading a file to GCS using Bifrost is as simple as calling the UploadFile method on the Bifrost client with the name of the bucket, the path to the file on your local machine, and the name of the file on GCS.
+
+## Shipping multiple files to Google Cloud Storage via the rainbow bridge
+If you want to upload multiple files using Bifrost with GCS, you can use the UploadMultiFile method provided by the GCS bridge in Bifrost. Here is an example code snippet:
+
+```go
+// Upload multiple files
+uploadedFiles, err := bridge.UploadMultiFile(bifrost.MultiFile{
+	Files: []bifrost.File{
+		{
+			Path:     "../shared/image/aand.png",
+			Filename: "a_and_ampersand.png",
+			Options: map[string]interface{}{
+				bifrost.OptMetadata: map[string]string{
+					"originalname": "aand.png",
+				},
+				bifrost.OptACL: bifrost.ACLPublicRead,
+			},
+		},
+		{
+			Path:     "../shared/image/hair.jpg",
+			Filename: "hair_of_opensaucerer.jpg",
+			Options: map[string]interface{}{
+				bifrost.OptMetadata: map[string]string{
+					"originalname": "hair.jpg",
+				},
+			},
+		},
+		{
+			Path:     "../shared/image/bifrost.webp",
+			Filename: "bifrost_bridge.webp",
+			Options: map[string]interface{}{
+				bifrost.OptMetadata: map[string]string{
+					"originalname": "bifrost.jpg",
+					"universe":     "Marvel",
+				},
+			},
+		},
+	},
+
+	// since we want both files to be public, we can set the global options rather than setting it for each file
+	// say 3 of 4 files need to share the same option, you can set globally for those 3 files and set the 4th file's option separately, bifrost won't override the option
+	GlobalOptions: map[string]interface{}{
+		bifrost.OptACL: bifrost.ACLPrivate,
+	},
+})
+if err != nil {
+	fmt.Println(err)
+	return
+}
+
+for _, file := range uploadedFiles {
+	fmt.Printf("Uploaded file: %s to %s\n", file.Name, file.Preview)
+}
+```
+
+## Additional Resources
+- [Google Cloud Storage Documentation](https://cloud.google.com/storage/docs)
+- [Bifrost GitHub Repository](https://github.com/opensaucerer/bifrost)

```

Comment:
But we don't need to put bifrost github repository here again....since they are within it. 

Rather...let's link these resources
- Setting up GCS
- Setting up gcloud CLI